### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by zi1zi** [clovaai/mxfont#25]
Thank you for your excellent work. I would like to inquire about the preparation of your dataset. My dataset consists of 30 fonts, each with 1000 characters. Among them, 20 fonts were used for training, with 800 characters. The test set consists of two parts: one part is the remaining 200 characters from the 20 fonts used in the training set, and the other part is the remaining 200 characters from the remaining 10 fonts. The source font is a font from the training set, May I ask how to place my dataset in the folder of your project, how to adjust training parameters, and how to reflect 800 training characters for each font in the project? I have seen that the training and val folders in your project are the same. Does the txt file list all the characters contained in the ttf for each font? Why should all the characters be listed? What is the purpose of this list of available characters? A question from a beginner in an industry
